qtpy/tests/test_qtquick.py

```diff
@@ -28,7 +30,7 @@ def test_qtquick():
         assert QtQuick.QSGFlatColorMaterial is not None
     assert QtQuick.QSGGeometry is not None
     assert QtQuick.QSGGeometryNode is not None
-    #assert QtQuick.QSGImageNode is not None
+    # assert QtQuick.QSGImageNode is not None
```

Comment:
Checking, seems like this assert needs to be inside the `if not PYSIDE2` validation, could you uncomment it and move it there please?